website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +76,147 @@ const columns = [
             }
         }
     },
+    {
+        field: 'minF&B',
+        headerName: 'Min F&B Per Contract',
+        description: 'F&B Minimum Per Contract',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['F&B Minimum Per Contract']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'area',
+        headerName: 'SQFT per Event',
```

Comment:
I do not like this text. It isn't clear to the user what you are communicating.

How does "Square Footage Utilized" sound?

Reply:
Yeah that sounds way better